### PR TITLE
[ADD] standard files, flake8

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+ - git clone https://github.com/gurneyalex/maintainer-quality-tools.git $HOME/maintainer-quality-tools
```
Comment (pedrobaeza) [inline .travis.yml]: Please change this route to the OCA (definitive) new one.

**Comment by bwrsandman** [comment]
> [Coverage Status](https://coveralls.io/builds/941111)

This might be because there are no tests and few python files.

**Comment by pedrobaeza** [comment]
Travis passes PEP8 test, so :+1:

**Comment by bwrsandman** [comment]
> This might be because there are no tests and few python files.

Nevermind...

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**